packages/@aws-cdk/aws-synthetics/lib/runtime.ts

```diff
@@ -134,6 +134,16 @@ export class Runtime {
    */
   public static readonly SYNTHETICS_NODEJS_PUPPETEER_3_5 = new Runtime('syn-nodejs-puppeteer-3.5', RuntimeFamily.NODEJS);
 
+  /**
+   * `syn-nodejs-puppeteer-3.6` includes the following:
+   * - Lambda runtime Node.js 14.x
+   * - Puppeteer-core version 10.1.0
+   * - Chromium version 92.0.4512
+   *
+   * @see https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Library_nodejs_puppeteer.html#CloudWatch_Synthetics_runtimeversion-nodejs-puppeteer-3.6
```

Comment:
I don't see this in the docs yet. Want to make sure we don't jump the gun and add this in before cloudformation supports this. That means we need to do add this to the existing integ test and run `yarn integ` to ensure that the new runtime is supported.